chart/openfaas/README.md

```diff
@@ -51,6 +51,18 @@ $ helm upgrade --install openfaas openfaas/ \
    --set functionNamespace=openfaas-fn
 ```
 
+## Activate basic auth for gateway
+
+Create a Kubernetes secret named `basic-auth` in `openfaas` namespace :
+
+```
+k create secret generic basic-auth --from-literal=user=test --from-literal=password=test
```

Comment:
```
-bash: k: command not found
```

I think `k` is not a standard alias for `kubectl`

Reply:
My bad it's my own alias. I will change it.

Reply:
Fixed in second commit. :)